Changes required by vhost

elmankku opened the pull request.

elmankku committed:
Add kernel modules required by vhost_net

Signed-off-by: Markku Ahvenjärvi <markkux@ssrc.tii.ae>

elmankku committed:
sel4-virt: Fix license warning

Signed-off-by: Markku Ahvenjärvi <markkux@ssrc.tii.ae>

elmankku committed:
vhost: add kernel configurations

Signed-off-by: Markku Ahvenjärvi <markkux@ssrc.tii.ae>

hlyytine reviewed (approved):
Looks good, maybe that -glx/+vhost could be in separate commits, tho.

elmankku committed:
qemu: enable vhost config

Signed-off-by: Markku Ahvenjärvi <markkux@ssrc.tii.ae>

elmankku committed:
qemu: conditionally enable display acceleration

Include opengl dependent package configs only if enabled on the distro.
Also update now non-existent 'glx' config with 'epoxy'.

Signed-off-by: Markku Ahvenjärvi <markkux@ssrc.tii.ae>

elmankku commented:
Done, and now conditionally enabling opengl dependent PACKAGE_CONFIGS.
